From: Hector Palacios <hector.palacios@digi.com>
Date: Wed, 14 Jan 2026 17:11:28 +0100
Subject: [PATCH] ccimx95: configure console on LPUART1

On version 2 of the SOM, the console has been changed
to LPUART1.

Upstream-Status: Inappropriate [DEY specific]

Signed-off-by: Hector Palacios <hector.palacios@digi.com>

https://onedigi.atlassian.net/browse/DEL-9915
---
 boards/ccimx95/config_board.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/boards/ccimx95/config_board.h b/boards/ccimx95/config_board.h
index ec97626a2a28..e666575ae6e8 100644
--- a/boards/ccimx95/config_board.h
+++ b/boards/ccimx95/config_board.h
@@ -2,6 +2,7 @@
 ** ###################################################################
 **
 ** Copyright 2023-2024 NXP
+** Copyright 2025-2026 Digi International Inc.
 **
 ** Redistribution and use in source and binary forms, with or without modification,
 ** are permitted provided that the following conditions are met:
@@ -52,7 +53,7 @@
 /* Defines */
 
 /*! Config for UART instance */
-#define BOARD_DEBUG_UART_INSTANCE    6U
+#define BOARD_DEBUG_UART_INSTANCE    1U
 /*! Config for UART baudrate */
 #define BOARD_DEBUG_UART_BAUDRATE    115200U
 
